Translations/UltraPunishments_German.lang

```diff
@@ -1,47 +1,47 @@
 #SERVER: Server
-#PUNISH: Punish
-#OR: or
-#REPORTS: Reports
-#GUI_OVERVIEW_REPORTS_ACTION: Click to show opened reports
-#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS_NONE: Player has no active reports
-#GUI_PLAYERVIEW_PUNISHMENTS_ACTIVE: Active Punishments
-#REPORT: Report
-#GUI_PLAYERVIEW_EXCLUDE_ACTION: Click to %1 punishments for this player
-#GUI_OVERVIEW_TEMPLATES_ACTION: Click to show templates
-#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Punishments for this player have been disabled.
-#TYPE_IN_MESSAGE: Type in a message
-#SET_A_DURATION: Click to set duration
-#PLAYERS: Players
-#REASON: Reason
-#TEMPLATES: Templates
-#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS: Active Reports by
-#EVIDENCE: Evidence
-#GUI_OVERVIEW_SETTINGS_ACTION: Click to show Settings
-#SET_A_REASON: Click to set reason
-#GUI_PLAYERVIEW_EXCLUDE_TITLE: Exclude from Punishments
-#MESSAGE: Message
-#GUI_PLAYERVIEW_PUNISHMENTS_ACTION: Click to show punishments
-#GUI_OVERVIEW_PLAYERS_ACTION: Click to show players
-#SETTINGS: Settings
-#TYPE_IN_REASON: Type in a reason
-#GUI_PLAYERVIEW_WARNINGS_ACTION: Click to show warnings
-#GUI_PLAYERVIEW_WARNINGS_NOWARNING: Player has never been warned
-#WARNINGS: Warnings
-#BAN: Ban
-#DURATION: Duration
-#GUI_OVERVIEW_TEMPLATES_MUTETEMPLATES: Mute Templates
-#GUI_PLAYERVIEW_PUNISH_LEFTACTION: **Left Click** to %1
-#PUNISHMENTS: Punishments
+#PUNISH: Bestrafen
+#OR: oder
+#REPORTS: Meldungen
+#GUI_OVERVIEW_REPORTS_ACTION: Klicke um offene Meldungen anzuzeigen
+#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS_NONE: Spieler hat keine aktiven Bestrafungen
+#GUI_PLAYERVIEW_PUNISHMENTS_ACTIVE: Aktive Bestrafungen
+#REPORT: Melden
+#GUI_PLAYERVIEW_EXCLUDE_ACTION: Klicke um %1 Bestrafungen für diesen Spieler
+#GUI_OVERVIEW_TEMPLATES_ACTION: Klicke um die Vorlagen anzuzeigen
+#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Bestrafungen für diesen Spieler wurden deaktiviert
+#TYPE_IN_MESSAGE: Gebe eine Nachricht ein
+#SET_A_DURATION: Klicke um die Dauer zu setzen
+#PLAYERS: Spieler
+#REASON: Grund
+#TEMPLATES: Vorlagen
+#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS: Aktive Meldungen von
+#EVIDENCE: Beweise
+#GUI_OVERVIEW_SETTINGS_ACTION: Klicke um die Einstellungen anzuzeigen
+#SET_A_REASON: Klicke um den Grund zu setzen
+#GUI_PLAYERVIEW_EXCLUDE_TITLE: Von Bestrafungen ausschließen
+#MESSAGE: Nachricht
+#GUI_PLAYERVIEW_PUNISHMENTS_ACTION: Klicke um die Bestrafungen anzuzeigen
+#GUI_OVERVIEW_PLAYERS_ACTION: Klicke um die Spieler anzuzeigen
+#SETTINGS: Einstellungen
+#TYPE_IN_REASON: Gebe einen Grund ein
```

Comment:
Change it to "Gib einen Grund ein"

Reply:
I think "Gebe einen Grund ein" is "better" language

Reply:
Gib ist die korrekte Befehlsform. Du kannst im Duden nachschauen. Bitte ändere es. :)